src/Aspire.Hosting.Azure.Storage/AzureStorageExtensions.cs

```diff
@@ -314,6 +370,22 @@ public static IResourceBuilder<AzureQueueStorageResource> AddQueues(this IResour
         return builder.ApplicationBuilder.AddResource(resource);
     }
 
+    /// <summary>
+    /// Allows setting the properties of an Azure Blob Storage container resource.
+    /// </summary>
+    /// <param name="builder">The Azure Blob Storage container resource builder.</param>
+    /// <param name="configure">A method that can be used for customizing the <see cref="AzureBlobStorageContainerResource"/>.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<AzureBlobStorageContainerResource> WithProperties(this IResourceBuilder<AzureBlobStorageContainerResource> builder, Action<AzureBlobStorageContainerResource> configure)
```

Comment:
I don't think we need this method right now. There are no properties to set on an `AzureBlobStorageContainerResource`.

Reply:
I'll remove it then :)